spring/boot-webflux-autoconfigure/src/main/java/com/linecorp/armeria/spring/web/reactive/ArmeriaServerHttpRequest.java

```diff
@@ -92,6 +92,11 @@ private static HttpHeaders fromArmeriaHttpHeaders(com.linecorp.armeria.common.Ht
         return cookies;
     }
 
+    @Override
+    protected String initId() {
+        return ctx.uuid().toString();
```

Comment:
Nice catch :+1:

Reply:
This was corrected with Minwoo help. :)  👍